src/main/java/de/chojo/repbot/dao/access/gdpr/GdprUser.java

```diff
@@ -160,9 +160,13 @@ public boolean sendData(ShardManager shardManager) {
                     .addFiles(FileUpload.fromData(tempFile.toFile()))
                     .complete();
         } catch (RuntimeException e) {
-            log.warn("Could not send gdpr data to user {}. File sending failed.", userId, e);
+            log.warn("Could not send gdpr data to user {}. File sending failed.", userId(), e);
```

Comment:
Do you really want to include the exception here? Also perhaps you should insert more information like the errorcode in the log message.

Reply:
Yeah message of the error might suffice.